config/ticket.php

```diff
@@ -6,4 +6,12 @@
 
     // Define your user model class for connect tickets to users.
     'user_model' => null,
+
+    'bucket' => 'public/ticket',
+
+    'dir_layer_number' => 2,
+
+    'attachment_rules' => [
+        'required', 'file', 'mimes:jpg,png,txt', 'mimetypes:text/plain,image/jpeg,image/png', 'max:1024'
```

Comment:
some of the rules are not configurable, they're just mandatory (like required or file). please separate them.
(extensions, max file sizes)